python-backend/app/commands.py

```diff
@@ -161,3 +163,15 @@ def delete_data():
             db.session.rollback()
             click.echo(f'Error, failed on commit.')
             raise
+
+    if app.config.get('ENVIRONMENT_NAME') == 'prod':
```

Comment:
Do we want a placeholder for if someone is not in prod, so they get a 'This method is only available in prod' to make it obvious, or are we ok with a basic 'command not found'?

Reply:
I see this has been discussed above , disregard.